scipy/misc/meson.build

```diff
@@ -0,0 +1,18 @@
+python_sources = [
+  '__init__.py',
+  'common.py',
+  'doccer.py',
+]
+
+py3.install_sources(
+    python_sources,
+    pure: false,
+    subdir: 'scipy/misc'
```

Comment:
minor: indentation level in `meson.build` files should be 2 spaces. 

we can fix this later in a style cleanup, that will be needed before we open a PR to SciPy for adding the full build system.

Reply:
Sure!, I will keep that in my mind next time